pkg/loki/modules.go

```diff
@@ -410,6 +410,8 @@ func (t *Loki) initQueryFrontend() (_ services.Service, err error) {
 	t.stopper = stopper
 
 	roundTripper = tripperware(roundTripper)
+	roundTripper = t.QueryFrontEndTripperware(roundTripper)
```

Comment:
Curious where is that set ?

Reply:
Shoulnd't this be something like:

```go
if t.QueryFrontEndTripperware != nil {
	roundTripper = t.QueryFrontEndTripperware(roundTripper)
}
```

Reply:
Good catch that would have crashed. The e2e test doesn't use the QueryFrontend.
This is set in the [initLabelAccessTripperware](https://github.com/grafana/loki-enterprise/pull/97/files#diff-c1c2d523cdbf7e8b96548805f225402fdbe41e0aac157c97e6d8c6413cbdf45eR174) method in GEL